components/SearchForm/SearchForm.vue

```diff
@@ -1,19 +1,26 @@
 <template>
   <div class="search-form" @keyup.enter="$emit('search')">
     <div class="input-wrap">
-      <input :value="value" @input="$emit('input', $event.target.value)" />
-      <button v-if="q" class="btn-clear-search" @click="$emit('clear')">
+      <input :value="value" placeholder="Enter search criteria" @input="$emit('input', $event.target.value)" />
+      <button v-if="q" class="btn-clear-search" @click="$emit('clear')">    
         <svg-icon
           name="icon-clear"
           stroke="red"
           color="#909399 #fff"
           height="22"
           width="22"
-        />
+      />
       </button>
     </div>
-    <el-button class="mr-8 btn-submit-search" @click="$emit('search')">
-      Search
+    <el-button class="btn-submit-search" @click="$emit('search')">
+      <svg-icon
+        icon="icon-magnifying-glass"
+        height="16"
+        width="16"
+      />
```

Comment:
Quick change on this, looking at the design, it should be mirrored in the other direction:
![image](https://user-images.githubusercontent.com/55994045/121201630-a9f90c80-c842-11eb-93e6-8872a9b66461.png)
Adding `dir="left"` should fix that. Also, I underestimated how small 16px would be, maybe we should try 25px for the `width` and `height`?